packages/application-generic/src/usecases/get-feature-flag/get-is-api-rate-limiting-enabled.use-case.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@nestjs/common';
```

Comment:
This file was also renamed from `get-is-request-rate-limiting-enabled` -> `get-is-api-rate-limiting-enabled`